### PR TITLE
Update upgrade_guide.md

### PR DESCRIPTION
**Pull request by vainkop**
Add How to upgrade CVAT from v2.2.0 to v2.3.0

### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added a description of my changes into [CHANGELOG](https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md) file
- [x] I have updated the [documentation](
  https://github.com/cvat-ai/cvat/blob/develop/README.md#documentation) accordingly
- [x] I have added tests to cover my changes
- [x] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [x] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

**Comment by vainkop** [comment]
@nmanovic please review. Thank you.

File: site/content/en/docs/administration/advanced/upgrade_guide.md [hunk 1]
```diff
@@ -53,25 +53,53 @@ To upgrade CVAT, follow these steps:
...
+export CVAT_VERSION="2.3.0"
+cd ..
+mv cvat cvat_220
+wget https://github.com/opencv/cvat/archive/refs/tags/v${CVAT_VERSION}.zip
+unzip v${CVAT_VERSION}.zip && mv cvat_${CVAT_VERSION} cvat
+cd cvat
+export CVAT_HOST=cvat.example.com
+export ACME_EMAIL=example@example.com
+docker pull cvat/server:v${CVAT_VERSION}
+docker tag cvat/server:v${CVAT_VERSION} openvino/cvat_server:latest
```
Comment (azhavoro) [inline site/content/en/docs/administration/advanced/upgrade_guide.md:75]: This is actual only for previous versions and doesn't matter for 2.2.0

File: site/content/en/docs/administration/advanced/upgrade_guide.md [hunk 2]
```diff
@@ -53,25 +53,53 @@ To upgrade CVAT, follow these steps:
...
+docker pull cvat/ui:v${CVAT_VERSION}
```
Comment (azhavoro) [inline site/content/en/docs/administration/advanced/upgrade_guide.md:76]: can be replaced with one command `docker compose pull`

File: site/content/en/docs/administration/advanced/upgrade_guide.md [hunk 3]
```diff
@@ -53,25 +53,53 @@ To upgrade CVAT, follow these steps:
...
+docker tag cvat/ui:v${CVAT_VERSION} openvino/cvat_ui:latest
```
Comment (azhavoro) [inline site/content/en/docs/administration/advanced/upgrade_guide.md:77]: the same comment

File: site/content/en/docs/administration/advanced/upgrade_guide.md [hunk 4]
```diff
@@ -53,25 +53,53 @@ To upgrade CVAT, follow these steps:
...
+docker-compose up -d cvat_db
+docker exec -i cvat_db psql -q -d postgres < ../cvat.db.dump
+docker-compose -f docker-compose.yml -f docker-compose.dev.yml -f docker-compose.https.yml up -d
```
Comment (azhavoro) [inline site/content/en/docs/administration/advanced/upgrade_guide.md:80]:
```suggestion
docker compose -f docker-compose.yml -f docker-compose.dev.yml -f docker-compose.https.yml up -d
```

File: site/content/en/docs/administration/advanced/upgrade_guide.md [hunk 5]
```diff
@@ -53,25 +53,53 @@ To upgrade CVAT, follow these steps:
...
+  
+## How to upgrade CVAT from v1.7.0 to v2.2.0.
+
+Step by step commands how to upgrade CVAT from v1.7.0 to v2.2.0.
 Let's assume that you have CVAT v1.7.0 working.
 ```shell
+export CVAT_VERSION="2.2.0"
 cd cvat
-docker compose down
+docker-compose down
```
Comment (azhavoro) [inline site/content/en/docs/administration/advanced/upgrade_guide.md:90]:
```suggestion
docker compose down
```

File: site/content/en/docs/administration/advanced/upgrade_guide.md [hunk 6]
```diff
@@ -53,25 +53,53 @@ To upgrade CVAT, follow these steps:
...
-docker pull cvat/server:v2.1.0
-docker tag cvat/server:v2.1.0 openvino/cvat_server:latest
-docker pull cvat/ui:v2.1.0
-docker tag cvat/ui:v2.1.0 openvino/cvat_ui:latest
-docker compose up -d
+docker pull cvat/server:v${CVAT_VERSION}
+docker tag cvat/server:v${CVAT_VERSION} openvino/cvat_server:latest
+docker pull cvat/ui:v${CVAT_VERSION}
+docker tag cvat/ui:v${CVAT_VERSION} openvino/cvat_ui:latest
+docker-compose up -d
```
Comment (azhavoro) [inline site/content/en/docs/administration/advanced/upgrade_guide.md:100]:
```suggestion
docker compose up -d
```

**Review by azhavoro — CHANGES_REQUESTED** [review]

File: site/content/en/docs/administration/advanced/upgrade_guide.md (hunk 2 above)
Comment (vainkop) [reply site/content/en/docs/administration/advanced/upgrade_guide.md:76]: Do you mean docker-compose pull? The dash is missing in the doc too.

File: site/content/en/docs/administration/advanced/upgrade_guide.md (hunk 1 above)
Comment (vainkop) [reply site/content/en/docs/administration/advanced/upgrade_guide.md:75]: Not sure how it's not actual as the doc clearly says `Note: updating CVAT from version 2.2.0 to version 2.3.0 requires additional manual actions with database data due to upgrading PostgreSQL base image major version. See details [here](https://github.com/opencv/cvat/blob/develop/site/content/en/docs/administration/advanced/upgrade_guide.md#how-to-upgrade-postgresql-database-base-image)

Comment (azhavoro) [reply site/content/en/docs/administration/advanced/upgrade_guide.md:75]: I mean only `docker tag ...` option

File: site/content/en/docs/administration/advanced/upgrade_guide.md (hunk 4 above)
Comment (vainkop) [reply site/content/en/docs/administration/advanced/upgrade_guide.md:77]: I've been using compose v1. This is the "new" v2 docker compose with a space, ok.

File: site/content/en/docs/administration/advanced/upgrade_guide.md [hunk 7]
```diff
@@ -24,7 +24,7 @@ To upgrade CVAT, follow these steps:
   If you have included [additional components](/docs/administration/basics/installation/#additional-components),
   include all compose configuration files that are used, e.g.:
   ```shell
-  docker compose -f docker-compose.yml -f components/analytics/docker-compose.analytics.yml down
+  docker compose -f docker compose.yml -f components/analytics/docker compose.analytics.yml down
```
Comment (nmanovic) [inline site/content/en/docs/administration/advanced/upgrade_guide.md:27]:
```suggestion
  docker compose -f docker-compose.yml -f components/analytics/docker-compose.analytics.yml down
```

**Review by nmanovic — APPROVED** [review]
@vainkop , looks great to me. Thanks for the contribution!

**Pull request merged**